### PR TITLE
Update bundling and pushing a theme.md

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-2089](https://jira.bigcommerce.com/browse/DEVDOCS-2089)

## What changed?
Updated per refactor IA guidelines

### CODE REVIEW

**Review by mikrdgz — CHANGES_REQUESTED** [review]
This all looks good to me. The only change needed, I think, is adding a reminder somewhere that users should switch to OAuth if possible

[https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#migrating-from-legacy-to-oauth](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#migrating-from-legacy-to-oauth)

File: docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md [hunk 1]
```diff
@@ -49,80 +49,80 @@ If you have added any new subdirectories or files to your base theme, verify tha
...
+BigCommerce provides two alternatives for uploading a theme to its associated BigCommerce store. You can perform either a control panel upload, or a command line upload. These options are available depending on how you've authorized your theme:
 
-### Control-Panel Upload (OAuth or Basic Auth)
+### Control panel upload (OAuth or Basic Auth)
 
-Uploading your theme via the BigCommerce control panel's GUI is compatible with any store token (whether OAuth or Basic-Auth), and with any OAuth token scope, and with all versions of Stencil CLI. However, this option requires that you first use stencil bundle to package your theme into a .zip file. (Prepare your file according to Bundle Only above.)
+Uploading your theme via the BigCommerce control panel's GUI is compatible with any store token (whether OAuth or Basic Auth), with any OAuth token scope, and with all versions of Stencil CLI. However, this option requires that you first use a stencil bundle to package your theme into a .zip file. (Prepare your file according to Bundle only above.)
```
Comment (traci707) [inline docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md:102]:
```suggestion
Uploading your theme via the BigCommerce control panel's GUI is compatible with any store token (whether OAuth or Basic Auth) and with any OAuth token scope, and with all versions of Stencil CLI. We recommend for users to use OAuth if possible. However, this option requires that you first use a stencil bundle to package your theme into a .zip file. (Prepare your file according to Bundle only above.)

```

File: docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md [hunk 2]
```diff
@@ -2,20 +2,20 @@
...
+- [Other bundling or upload errors](#other-bundling-or-upload-errors)
 - [Resources](#resources)
 
 </div> 
 
-## Confirm Dependencies
+## Confirm dependencies
 
-If you have customized a theme originally downloaded from the BigCommerce Theme Marketplace: Before you package your theme, make sure your theme directory includes all the dependencies that BigCommerce requires for submission.
+If you have customized a theme initially downloaded from the BigCommerce Theme Marketplace: Before you package your theme, make sure your theme directory includes all the dependencies that BigCommerce requires for submission.
```
Comment (uwikaiddi) [inline docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md:18]:
```suggestion
If you have customized a theme originally downloaded from the BigCommerce Theme Marketplace: Before you package your theme, make sure your theme directory includes all the dependencies that BigCommerce requires for submission.

```

File: docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md (hunk 1 above)
Comment (uwikaiddi) [inline docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md:102]:
```suggestion
Uploading your theme via the BigCommerce control panel's GUI is compatible with any store token (whether OAuth or Basic Auth), with any OAuth token scope, and with all versions of Stencil CLI. However, this option requires that you first use a stencil bundle to package your theme into a zip file. Prepare your file according to the instructions in the [Bundle only](#bundle-only) section.

```

File: docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md [hunk 3]
```diff
@@ -49,80 +49,80 @@ If you have added any new subdirectories or files to your base theme, verify tha
...
 
-### Software Requirements / Resolving Lint Errors
+### Software requirements / resolving lint errors
 
 * Only use the `stencil bundle` or the `stencil push` command to process and package themes for submission. These commands generate `.zip` files that match BigCommerce's expected structure. They also generate metadata required for your theme to function properly.
 
 * Do not create `.zip` files using general-purpose archiving software. The resulting files will trigger errors upon upload to BigCommerce.
 
-* Do not open a bundled theme `.zip` file to add, delete, rename, or update files. Doing so will make your theme unusable on the production store.
+* Do not open a bundled theme `.zip` file to add, delete, rename, or update files. Doing so will make your theme unusable in the production store.
```
Comment (uwikaiddi) [inline docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md:92]:
```suggestion
* Do not open a bundled theme zip file to add, delete, rename, or update files. Doing so will make your theme unusable in the production store.

```

File: docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md [hunk 4]
```diff
@@ -49,80 +49,80 @@ If you have added any new subdirectories or files to your base theme, verify tha
...
 
-Check the resulting `.zip` file's size before you proceed. The zipped bundle should be only a few megabytes. BigCommerce imposes a hard limit of 50 MB, and any file size approaching that is problematic. If your `.zip` file fits comfortably within the size limit above, jump directly to [Pushing Your Theme](https://developer.bigcommerce.com/stencil-docs/deploying-a-theme/bundling-and-pushing#pushing-your-theme). However, if your `.zip` file approaches or exceeds 50 MB, you must first use one of these procedures to restructure your theme to a size that's manageable for upload to BigCommerce:
+Check the resulting `.zip` file's size before you proceed. The zipped bundle should be only a few megabytes. BigCommerce imposes a hard limit of 50 MB, and any file size approaching that is problematic. If your `.zip` file fits comfortably within the size limit above, jump directly to [Pushing Your Theme](https://developer.bigcommerce.com/stencil-docs/deploying-a-theme/bundling-and-pushing#pushing-your-theme). However, if your `.zip` file approaches or exceeds 50 MB, you must first use one of these procedures to restructure your theme to a manageable size for upload to BigCommerce:
```
Comment (uwikaiddi) [inline docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md:73]:
```suggestion
Check the resulting zip file's size before you proceed. The zipped bundle should be only a few megabytes. BigCommerce imposes a hard limit of 50 MB, and any file size approaching that is problematic. If your zip file fits comfortably within the size limit above, jump directly to [Pushing Your Theme](https://developer.bigcommerce.com/stencil-docs/deploying-a-theme/bundling-and-pushing#pushing-your-theme). However, if your zip file approaches or exceeds 50 MB, you must first use one of these procedures to restructure your theme to a manageable size for upload to BigCommerce:

```
Style guide: [Referring to file types](https://intranet.bigcommerce.com/display/SST/Naming+Guidelines#NamingGuidelines-Referringtofiletypes)

File: docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md [hunk 5]
```diff
@@ -49,80 +49,80 @@ If you have added any new subdirectories or files to your base theme, verify tha
...
 
 </div>
 </div>
 </div>
 
-## Bundling Your Theme
+## Bundling your theme
 
-Once you have verified the requirements above, you are ready to process and package your theme for upload to BigCommerce. Stencil CLI provides two options for creating a `.zip` file that contains all of your theme's essentials, while excluding redundant components. The options are either only bundling your theme, or bundling and pushing your theme. These options are available depending on how you've [authorized](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/troubleshooting-your-setup#stencil-start-errors) your theme:
+Once you have verified the requirements above, you are ready to process and package your theme for upload to BigCommerce. Stencil CLI provides two options for creating a `.zip` file that contains all of your theme's essentials while excluding redundant components. The options are either only bundling your theme, or bundling and pushing your theme. These options are available depending on how you've [authorized](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/troubleshooting-your-setup#stencil-start-errors) your theme:
```
Comment (uwikaiddi) [inline docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md:61]:
```suggestion
Once you have verified the requirements above, you are ready to process and package your theme for upload to BigCommerce. Stencil CLI provides two options for creating a zip file that contains all of your theme's essentials while excluding redundant components. The options are either only bundling your theme, or bundling and pushing your theme. These options are available depending on how you've [authorized](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/troubleshooting-your-setup#stencil-start-errors) your theme:

```

File: docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md [hunk 6]
```diff
@@ -49,80 +49,80 @@ If you have added any new subdirectories or files to your base theme, verify tha
...
-For the upload steps in the control panel, see the [Uploading Custom Themes](https://support.bigcommerce.com/s/article/Stencil-Themes#download-upload) article in our Knowledge Base.
+For the control panel's upload steps, see the [Uploading Custom Themes](https://support.bigcommerce.com/s/article/Stencil-Themes#download-upload) article in our Knowledge Base.
 
 For error codes that you might encounter when uploading a theme – and corresponding workarounds, see [Troubleshooting Theme Uploads](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/troubleshooting-your-setup).
 
-### Command-Line Upload (OAuth Required)
+### Command line upload (OAuth required)
 
-The stencil push command allows you to both bundle your theme and upload it to the store, with a single terminal command. To run stencil push, you must first:
+The stencil push command allows you to bundle your theme and upload it to the store, with a single terminal command. To run stencil push, you must first:
```
Comment (uwikaiddi) [inline docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md:110]:
```suggestion
The `stencil push` command allows you to bundle your theme and upload it to the store, with a single terminal command. To run `stencil push`, you must first:

```

File: docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md [hunk 7]
```diff
@@ -49,80 +49,80 @@ If you have added any new subdirectories or files to your base theme, verify tha
...
 
-Successfully initialize your theme using an OAuth token that was created with the Themes:modify scope.
+Successfully initialize your theme using an OAuth token that you created with the Themes: modify scope.
```
Comment (uwikaiddi) [inline docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md:112]:
```suggestion
Successfully initialize your theme using an OAuth token that you created with the `Themes: modify scope`.

```

File: docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md [hunk 8]
```diff
@@ -133,13 +133,13 @@ Stencil CLI will display `ok` confirmations, or `not ok` errors, or `warnings` f
...
+### Successful upload
 
-Upon a successful upload, you will be prompted: `Would you like to apply your theme to your store at <storehash>? (y/N)` Any response except `y` or `Y` will be processed as "No." You can always apply the theme later through the control panel.
+Upon successful upload, you will receive the prompt, `Would you like to apply your theme to your store at <storehash>? (y/N)`. Any response except `y` or `Y` processes as "No." You can always apply the theme later through the control panel.
 
-### Apply Which Variation?
+### Apply which variation?
 
-If you chose to apply the newly uploaded theme, you will be prompted: `Which variation would you like to apply? (Use arrow keys)`
+If you chose to apply the newly uploaded theme, you will receive the prompt: `Which variation would you like to apply? (Use arrow keys)`
```
Comment (uwikaiddi) [inline docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md:142]:
```suggestion
If you chose to apply the newly uploaded theme, you will receive the prompt: "Which variation would you like to apply? (Use arrow keys)"

```

File: docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md [hunk 9]
```diff
@@ -159,9 +159,9 @@ Stencil CLI will then confirm which variation is active on the storefront.
 
 ![](//s3.amazonaws.com/user-content.stoplight.io/6116/1539055915081 "")
 
-## Theme Quota Warning
+## Theme quota warning
 
-If you run `stencil push` when your store's `My Themes` section has reached its maximum of 20 themes, you will be prompted to select at least one existing theme for deletion.
+If you run `stencil push` when your store's `My Themes` section has reached its maximum of 20 themes, you will receive a prompt to select at least one existing theme for deletion.
```
Comment (uwikaiddi) [inline docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md:164]:
```suggestion
If you run `stencil push` when your store's My Themes section has reached its maximum of 20 themes, you will receive a prompt to select at least one existing theme for deletion.

```

File: docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md [hunk 10]
```diff
@@ -174,30 +174,30 @@ Custom themes – which are available for selection – will have a circle to th
 
 Use your arrow keys to move the selection caret to each theme/version that you want to select. Then press the spacebar to select it. (Filled circles will indicate your selected themes/versions.)
 
-If you are certain of your selections, you can next press `Enter` to delete the themes.
+If you are certain of your selections, you can press `Enter` to delete the themes.
```
Comment (uwikaiddi) [inline docs/stencil-docs/deploying-a-theme/bundling-and-pushing.md:177]:
```suggestion
If you are certain of your selections, you can press **Enter** to delete the themes.

```

**Review by uwikaiddi — CHANGES_REQUESTED** [review]

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**